hphp/hack/src/errors/typing_error.ml

```diff
@@ -4342,14 +4343,26 @@ module Primary = struct
       lazy [(pos2, "Mixed with this kind of field (key => value)")],
       [] )
 
-  let unbound_name_typing pos name =
+  let unbound_name_typing pos name class_exists =
```

Comment:
I think this error occurs for any unknown name, not just function calls. Could you add a test for `$x = Person;` that we don't offer `new`?